### PR TITLE
[native, managed] Add Calc example

### LINKED ISSUE #23: Can you provide examples of C++ and C#?
Can you provide examples of C++ and C#?The test is too simpler，i don't know how to use it

### PR DESCRIPTION
**Pull request by ara-ayvazyan**
Resolves #23

### CODE REVIEW

File: Examples/Calc/CalcNative/Service.h [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            {
+                std::cout << "Service destructed" << std::endl;
+            }
+        }
+
+        template <typename Callback>
+        void operator()(const Request& request, Callback&& callback) const
+        {
+            Response response;
+            std::stringstream text;
```
Comment (Lyr1k) [inline Examples/Calc/CalcNative/Service.h]: minor: std::string?

Comment (ara-ayvazyan) [reply Examples/Calc/CalcNative/Service.h]: Need some basic formatting (don't want to use `std::to_string` here).

File: Examples/Calc/CalcNative/Service.h [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                text << '-';
+                break;
+
+            case Operation::Multiply:
+                response.Z = request.X * request.Y;
+                text << '*';
+                break;
+
+            case Operation::Divide:
+                response.Z = request.X / request.Y;
```
[the remark this answers is not in the document]
Comment (ara-ayvazyan) [reply Examples/Calc/CalcNative/Service.h]: Yeah, it won't crash, the rest not much important.

File: Examples/Calc/CalcNative/Client.cpp [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+            }
+
+            try
+            {
+                auto response = futureResponse.get();
+
+                const auto end = std::chrono::steady_clock::now();
+                const auto latency = std::chrono::duration_cast<std::chrono::microseconds>(end - begin);
+
+                std::cout << response.Text << response.Z << " [" << latency.count() << "us]" << std::endl;
```
[the remark this answers is not in the document]
Comment (ara-ayvazyan) [reply Examples/Calc/CalcNative/Client.cpp]: The `Text` already comes with space. I could move that part here.

**Pull request merged**